containers/codespaces-linux/.devcontainer/devcontainer.json

```diff
@@ -1,9 +1,9 @@
 {
-	"name": "VS Online (Linux Universal)",
+	"name": "Codespaces (Linux Universal)",
 	"dockerFile": "Dockerfile",
-	"remoteUser": "vsonline",
-	"workspaceMount": "source=${localWorkspaceFolder},target=/home/vsonline/workspace,type=bind,consistency=cached",
-	"workspaceFolder": "/home/vsonline/workspace",
+	"remoteUser": "codespace",
+	"workspaceMount": "source=${localWorkspaceFolder},target=/home/codespace/workspace,type=bind,consistency=cached",
+	"workspaceFolder": "/home/codespace/workspace",
 	"runArgs": [ "--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined" ],
 
 	// Add the IDs of extensions you want installed when the container is created.
```

Comment:
Is the extension  ms-vsonline.vsonline going to be renamed?

Reply:
Yes but not yet, so its going to be a different change and needs a little coordination.